data-prepper-plugins/common/src/main/java/com/amazon/dataprepper/plugins/certificate/model/Certificate.java

```diff
@@ -2,6 +2,7 @@
 
 import static java.util.Objects.requireNonNull;
 
+// TODO: accommodate encrypted private key with password
```

Comment:
Do we have an issue in GitHub to ensure we get to this?

Reply:
Will open one